group.go

```diff
@@ -51,12 +51,11 @@ type Scalar interface {
 	// Set to a fresh random or pseudo-random scalar
 	Pick(rand cipher.Stream) Scalar
 
-	// SetBytes sets the scalar from a big-endian byte-slice,
+	// SetBytes sets the scalar from a byte-slice,
 	// reducing if necessary to the appropriate modulus.
+	// The endianess of the byte-slice is determined by the
+	// implementation.
 	SetBytes([]byte) Scalar
```

Comment:
@jeffallen I remember you suggested that maybe we don't need `SetBytes` in the interface because it's only really needed in one place, eddsa I believe, the other places can be replaced with `Pick`. Is there a decision on that?

Reply:
Let me recap the story:
When `edwards25519.Scalar` were using a `big.Int` (well a `mod.Int` to be exact) for its computation, it was not possible to have an EdDSA implementation because of the "automatic" modulo. That is the reason we added a `SetBytes()` method (and its counterpart `Bytes` which turned out to be useless). 
Since a few months already, Bryan ported the usual constant time implementation to `edwards25519.Scalar` , so that now it only uses a `[]byte`. I *think* (notice that uncertainty here) that with this change, the need for `SetBytes` (/`Bytes`) can be alleviated, i.e. we can remove those. Need for some tests to confirm that.

That's one way, Another way, which is easier but faster, but still correct and "good", is also to correct the doc as @jeffallen does here :)